### ISSUE TITLE
Documentation page for file type support.

### ISSUE THREAD

**Issue by rosiel** [Islandora/documentation#1786]
I don't know if/how we support Tiff files. I think we should have a page of file types and an up to date explanation of what we can do with them.

**Comment by seth-shaw-unlv** [comment]
I agree this would be helpful. Drupal's media types are centered around types of content browsers can display. Ergo, Tiffs should be uploaded as File media, **not Image media**, due to insufficient browser support which, I imagine, is counter-intuitive for new users. _Islandora_ supports Tiffs in that we can both convert them into web-friendly formats or use large image viewers (like Open Seadragon), but this is something _we_ provide.
